app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
```

Comment:
- [ ] Good job 👏🏻 however the views in devise for the registration and login pages should match the provided wireframes, kindly make sure to implement this so your project is completed. 💯 👍🏻 

|   |   |
|-|-|
|![image](https://user-images.githubusercontent.com/79058364/224463299-a143df49-a75a-4f6b-9950-3fec01680cb4.png)|![image](https://user-images.githubusercontent.com/25227022/216383255-33d4fd58-29e6-417b-9b68-f97b4f6597dd.png)|